fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -856,6 +857,9 @@ private synchronized void logEdit(short op, Writable writable) {
 
         long start = System.currentTimeMillis();
 
+        Preconditions.checkState(Catalog.getCurrentCatalog().isMaster(),
```

Comment:
it will be crash when fe process start.

Reply:
I set the feType to MASTER before any writes, and reset the feType to oldType if transferToMaster failed. It will resolve this problem.